df_chat/drf/serializers.py

```diff
@@ -249,5 +249,5 @@ class UserNameSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = User
-        read_only_fields = ("display_name",)
```

Comment:
This was breaking the Swagger UI.

Reply:
Right. There is no "display_name" on the User model. Could you please replace it with "username"?

Reply:
Done!